content/articles/stream-filter-in-java-using-lambda-expressions/index.md

```diff
@@ -1,25 +1,29 @@
 ### Introduction
-Stream operations such as create and destroy are sandwiched by the filter() function (). In this case, it produces a Stream with elements from the input stream that meet the supplied predicate in the input stream. The filter() argument must be a stateless predicate that evaluates each component in the stream to assess whether or not it can be included.
+
+Stream operations such as create and destroy are sandwiched by the `filter()` function. In this case, it produces a `Stream` with elements from the input stream that meet the supplied predicate in the input stream. The filter() argument must be a stateless predicate that evaluates each component in the stream to assess if it can be included.

```

Comment:
This intro doesn't do a good job of telling the audience the significance of this technology. For an introduction, you should give context about why they should want/need to learn what you are about to teach and its significance